app/grandchallenge/algorithms/urls.py

```diff
@@ -31,6 +32,11 @@
 urlpatterns = [
     path("", AlgorithmList.as_view(), name="list"),
     path("create/", AlgorithmCreate.as_view(), name="create"),
+    path(
+        "create-for-phase/<phase_pk>/",
+        PhaseAlgorithmCreate.as_view(),
+        name="create-for-phase",
+    ),
```

Comment:
I think it would be nicer to have this view in the evaluation app, maybe at `"<slug>/algorithms/create/"`. We shouldn't expose phase pk's, this is better done through the slug and challenge.